includes/wc-core-functions.php

```diff
@@ -157,7 +157,7 @@ function wc_update_order( $args ) {
  * @param string $name Template name (default: '').
  */
 function wc_get_template_part( $slug, $name = '' ) {
-	$cache_key = sanitize_key( implode( '-', array( 'template-part', $slug, $name ) ) );
+	$cache_key = sanitize_key( implode( '-', array( 'template-part', $slug, $name ) ) . '_' . WC()->version );
```

Comment:
Thanks for the update. Sorry for nitpicking, but is there a reason to not add the version to the array like the rest of cache_key parts?